### PR TITLE
docs(mesh): add instruction how to update a license

### PR DESCRIPTION
**Pull request by jakubdyszkiewicz**
Closes https://github.com/Kong/docs.konghq.com/issues/5303

### Description

We are missing description how to update a license.

### Testing instructions

Netlify link: 

### Checklist 

- [X] Review label added 
- [X] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_src/mesh/plan-and-deploy/license.md
Comment (cloudjumpercat) [inline app/_src/mesh/plan-and-deploy/license.md:132]: 1. Update the `kong-mesh-license` Secret in the `kong-mesh-system` namespace with a new license:
```
kuma:
    controlPlane:
      secrets:
        - Env: "KMESH_LICENSE_INLINE"
          Secret: "kong-mesh-license"
          Key: "new-license.json"
```

Comment (cloudjumpercat) [reply app/_src/mesh/plan-and-deploy/license.md:132]: Not sure if this would be accurate? GitHub wouldn't let me add a suggestion without ruining the formatting.

File: app/_src/mesh/plan-and-deploy/license.md
Comment (cloudjumpercat) [inline app/_src/mesh/plan-and-deploy/license.md:133]:
```suggestion
1. Restart the control plane:
```

File: app/_src/mesh/plan-and-deploy/license.md
Comment (cloudjumpercat) [inline app/_src/mesh/plan-and-deploy/license.md:142]:
```suggestion
1. Update the license by doing one of the following:
  - If you used `KMESH_LICENSE_PATH`, update the content of the `values.yaml` file.
  - If you used `KMESH_LICENSE_INLINE`, update the value of the environment variable in the `values.yaml` file.
```
would this be accurate?

File: app/_src/mesh/plan-and-deploy/license.md
Comment (cloudjumpercat) [inline app/_src/mesh/plan-and-deploy/license.md:143]:
```suggestion
1. Restart the control plane:
```

**Review by cloudjumpercat — COMMENTED** [review]
Left a few suggestions on the page! Let me know if you have any questions and thanks for making these updates!

**Comment by cloudjumpercat** [comment]
@jakubdyszkiewicz Hi! I left a few suggestions on this PR, if you could take a look when you have the time. Thanks!

**Comment by slonka** [comment]
@cloudjumpercat is OOO, I'll take a look at this

File: app/_src/mesh/plan-and-deploy/license.md
Comment (slonka) [reply app/_src/mesh/plan-and-deploy/license.md:142]: Not really, there is no `values.yaml` in Universal mode because it's helm.

File: app/_src/mesh/plan-and-deploy/license.md
Comment (slonka) [reply app/_src/mesh/plan-and-deploy/license.md:143]: It's universal, there is no `kubectl`.

File: app/_src/mesh/plan-and-deploy/license.md
Comment (slonka) [reply app/_src/mesh/plan-and-deploy/license.md:133]: I'll do this manually.

File: app/_src/mesh/plan-and-deploy/license.md
Comment (slonka) [reply app/_src/mesh/plan-and-deploy/license.md:132]: We could have:

```
kubectl edit secrets kong-mesh-license -n kong-mesh-system
```

**Comment by slonka** [comment]
@cloudjumpercat - let me know if my changes make sense :)

File: app/_src/mesh/plan-and-deploy/license.md
Comment (cloudjumpercat) [inline app/_src/mesh/plan-and-deploy/license.md:132]:
```suggestion
1. Update the `kong-mesh-license` Secret in `kong-mesh-system` namespace with a new license:
```

**Review by cloudjumpercat — APPROVED** [review]
LGTM! I'll let you merge this when you're ready @slonka, since I don't think this is for the 2.2 release/any other Mesh release, but just wanted to be sure.

**Pull request merged**
